lib/paths.cf

```diff
@@ -315,7 +315,7 @@ bundle common paths
       "path[tr]"            string => "/usr/bin/tr";
       "path[wc]"            string => "/usr/bin/wc";
       "path[wget]"          string => "/usr/bin/wget";
-      "path[realpath]"      string => "/usr/bin/realpath";
+      "path[realpath]"      string => "/usr/bin/readlink -e";
```

Comment:
After thinking about it, you're trying to do two things in one:

* provide a path to a common binary
* provide a way to find the real path of something

The `paths` bundle **only** does the first thing. So I don't think it should lie about `realpath`.

For the second item, I would provide a bundle in `files.cf` **or** extend the `filestat()` function (but also note it already does quite a bit, see https://docs.cfengine.com/docs/master/reference-functions-filestat.html ). That would not only address the need, but it would also serve all platforms in a consistent way.